test/app/src/form.ts

```diff
@@ -45,6 +45,10 @@ const Form = `
     <option value="strict">Strict</option>
   </select><br/>
   <label for="_siwVersion">Sign-in Widget version (leave blank for bundled version)</label><input id="_siwVersion" name="_siwVersion" type="text" /><br/>
+  <label for="_siwAuthClient">Use authClient option?</label><br/>
+  <input id="authclient-on" name="_siwAuthClient" type="radio" value="true"/>YES (inject current instance)<br/>
```

Comment:
authclient here is lowercaseunspaced but forceRedirect below is camelCase (both mixed with kebab-case).  You're killing me :)